newIDE/app/src/UI/Theme/DarkTheme/Mosaic.css

```diff
@@ -23,3 +23,20 @@
 .mosaic-gd-dark-theme.mosaic.mosaic-blueprint-theme .mosaic-preview .mosaic-window-toolbar.draggable:hover {
   background: #3c4698 !important;
 }
+
+/* Separator draggable background */
+.mosaic-gd-dark-theme .mosaic-split.-column .mosaic-split-line {
+  height: 2px;
+  border-bottom: 1px solid #888888 !important;
+  border-top: 1px solid #888888 !important;
+  width: 8%;
+  margin: auto;
+  background-color: #444444;
```

Comment:
Does it look the same if you remove the background-color entirely? The less color we have in CSS, the more flexible the theme is :)

Reply:
background-color: #444444;
Oh yes color should set by variable, color is not very relevant indeed!